scripts/contributor_dashboard_debug.py

```diff
@@ -0,0 +1,257 @@
+# coding: utf-8
```

Comment:
Please make this file owned by the contributor experience team instead as discussed in https://github.com/oppia/oppia/pull/15857#discussion_r953070146

Reply:
Changed the owner.